src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocMethodCheck.java

```diff
@@ -701,12 +701,29 @@ private static int calculateTagColumn(Matcher javadocTagMatcher,
             final List<DetailAST> throwLiterals = findTokensInAstByType(blockAst,
                     TokenTypes.LITERAL_THROW);
             for (DetailAST throwAst : throwLiterals) {
-                final DetailAST newAst = throwAst.getFirstChild().getFirstChild();
-                if (newAst.getType() == TokenTypes.LITERAL_NEW) {
-                    final FullIdent ident = FullIdent.createFullIdent(newAst.getFirstChild());
-                    final ExceptionInfo exceptionInfo = new ExceptionInfo(
-                            createClassInfo(new Token(ident), currentClassName));
-                    returnValue.add(exceptionInfo);
+                DetailAST ancestor = throwAst;
```

Comment:
```
DetailAST ancestor = throwAst;
```
-->
```
DetailAST ancestor = throwAst.getParent();
```
We do not need to check `throw` AST itself in `while` as I understand

Reply:
Makes sense. Changed and will update regression soon.